### PR TITLE
Panel-table on case-page initially renders default panel in first row

### LINKED ISSUE #4526: Make default panels easier to access / overview
This is a quality of life feature requested by our geneticists.

In the Gene panels section of the case page, they often have a large number of optional panels.

The default panels are "hidden" among these. They are nicely highlighted, but still requires frequent scrolling to access.

Would it be possible to make accessing these even easier?

They suggested putting them at the top of the list, but one could also imagine a checkbox to toggle "show only default panels" or separating them out into a separate list.

Here is the Gene panels list for the demo case. Our geneticists commonly have 100+ panels here.

### PR DESCRIPTION
**Pull request by TereseBo**
This PR adds a functionality by replacing the default order of panels in the panel-table on case-page. This PR contains changes to render the panels in the table with the default panel in the first row of the table, followed by all other panels for the case in alphabetical order. 

In addition the option to sort the table depending on if the panel is the default or not has been added.

This PR closes  #4526

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. Fist book your testing time using the Pax software available at [https://pax.scilifelab.se/](https://pax.scilifelab.se). The resource you are going to call dibs on is `scout-stage` and the server is `cg-vm1`.
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
1. After testing is done, repeat procedure at [https://pax.scilifelab.se/](https://pax.scilifelab.se), which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

<details>
<summary>Testing on hasta server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. `ssh <USER.NAME>@hasta.scilifelab.se`
1. Book your testing time using the Pax software. `us; paxa -u <user> -s hasta -r scout-stage`. You can also use the WSGI Pax app available at [https://pax.scilifelab.se/](https://pax.scilifelab.se).
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `conda activate S_scout; pip freeze | grep scout-browser`
1. Deploy the branch to test: `bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_scout -t scout -b <this_branch>`
1. Make sure the branch is deployed: `us; scout --version`
1. After testing is done, repeat the `paxa` procedure, which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

**How to test**:
1. how to test it, possibly with real cases/data

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [ ] code approved by
- [x] tests executed by TB

### CODE REVIEW

**Comment by TereseBo** [comment]
On main branch the panels-table has 3 columns and is initially sorted in alphabetical order.

image

**Comment by TereseBo** [comment]
On this branch the panels-table has four columns and is initially rendered with the default panel first, otherwise in alphabetical order. The table can still be reordered by clicking the icon in each header.

image

File: scout/server/blueprints/cases/templates/cases/gene_panel.html [hunk 1]
```diff
@@ -7,6 +7,7 @@
         <thead class="table-light thead">
           <tr style="cursor: pointer; white-space: nowrap">
             <th>Panel <i class="fas fa-sort" data-bs-toggle="tooltip" title="Sort by gene panel name"></i></th>
+            <th><i class="fas fa-bars float-end mx-4" data-bs-toggle="tooltip" title="Sort by default panel"></i></th>
```
Comment (northwestwitch) [inline scout/server/blueprints/cases/templates/cases/gene_panel.html:10]:
```suggestion
            <th><i class="fas fa-sort float-end mx-4" data-bs-toggle="tooltip" title="Sort by default panel"></i></th>
```

Comment (northwestwitch) [reply scout/server/blueprints/cases/templates/cases/gene_panel.html:10]: Works fine, but perhaps use fa-sort to make is similar to the other columns?

**Review by northwestwitch — APPROVED** [review]
Works fine! 💯 
I have just a small style suggestion

File: scout/server/blueprints/cases/templates/cases/gene_panel.html (hunk 1 above)
Comment (TereseBo) [reply scout/server/blueprints/cases/templates/cases/gene_panel.html]: image

Updated icon and added a header as well as removed some very specific styling of the default column

**Pull request merged**